_search-plugins/hybrid-search.md

```diff
@@ -1018,4 +1018,191 @@ The response contains the matching documents that are listed after the `7yaM4JAB
         ]
     }
 }
-```
\ No newline at end of file
+```
+
+## Explainability in hybrid query
+**Introduced 2.19**
+{: .label .label-purple }
+
+The explainability feature helps you understand how scores are calculated, normalized, and combined in hybrid queries. When enabled, it provides detailed information about the scoring process for each search result. This includes the various score normalization techniques used (such as `min_max` or `rrf`), how different scores are combined, and the details of individual score calculations from sub-queries (like [Okapi BM25](https://en.wikipedia.org/wiki/Okapi_BM25) for text-based queries). This comprehensive insight into the scoring process makes it easier to understand and optimize your hybrid query results. For more information about explain, see corresponding [API reference page]({{site.url}}{{site.baseurl}}/api-reference/explain). 
```

Comment:
`This includes the various score normalization techniques used (such as `min_max` or `rrf`), how different scores are combined, and the details of individual score calculations from sub-queries (like [Okapi BM25](https://en.wikipedia.org/wiki/Okapi_BM25) for text-based queries). `

Can this sentence be little less wordy? I found a hard time in going through it and understand it completely. Now as I already have all the context so I was able to understand. But for CX it is a very wordy sentence will make there attention loose. WDYT?

Reply:
I can make that sentence simpler and drop some of the details, something like:
```
The explainability feature provides detailed information about the scoring process for each search result. It reveals the score normalization techniques used, how different scores are combined, and the calculations for individual sub-query scores.
```

Reply:
After I looked closer I've changed the whole paragraph to a simpler form. Hope that addressed concerns